resources/css/styles.css

```diff
@@ -229,7 +229,7 @@ input[type="text"]:focus, input[type="password"]:focus, select.form-control:focu
 .form-range::-webkit-slider-runnable-track {
     width: 100%;
     height: 10px;
-    background: linear-gradient(to right, #4a47a3 0%, #4a47a3 50%, #ddd 50%);
+    background: linear-gradient(to right, #4a47a3 calc(var(--value) * 100%), #ddd 100%, #ddd 100%);
```

Comment:
This will alone fix the issue. its interesting that adding `calc(var(--value) * 100%)` will fix the issue in chrome and removing this try to use any percentage, then the dynamic range is not updated even though I'm not setting the `--value`.  

No idea whats happening in here 😅😅😅